pkgs/tools/admin/afterburn/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, openssl, pkg-config, fetchCrate, rustPlatform }:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  version = "4.1.3";
+  pname = "afterburn";
+
+  src = fetchCrate {
+    crateName = pname;
+    inherit version;
+    sha256 = "0v233g8z9v37ks61gag3949c4ffrf6dns5jpwkfpihsjkyzd0p10";
+  };
+
+  cargoSha256 = "0503qs6bxqiacllx99kdkvwl01qxi5bfijgf088km24ys65aisgh";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/coreos/cargo";
+    description = "This is a small utility, typically used in conjunction with Ignition, which reads metadata from a given cloud-provider and applies it to the system.";
+    license = stdenv.lib.licenses.asl20;
```

since you have the `with stdenv.lib;` on line 20, these arent needed
```suggestion
    license = licenses.asl20;
```

---

Done!